ExposureNotification.App/ExposureNotification.App/App.xaml.cs

```diff
@@ -20,10 +21,18 @@ public App()
 			// Local Notification tap event listener
 			NotificationCenter.Current.NotificationTapped += OnNotificationTapped;
 
-			// Initialize the library which schedules background tasks, etc
+			// Initialize the library
 			Xamarin.ExposureNotifications.ExposureNotification.Init();
 
 			MainPage = new AppShell();
+
+			_ = InitializeBackgroundTasks();
```

Comment:
It is a bad practice to not `await` a `Task` because a non-awaited Task` `doesn't rethrow exceptions. I go into more detail [here](https://github.com/brminnick/AsyncAwaitBestPractices#explaination).

The easiest fix is to change `async Task InitializeBackgroundTasks()` to `async void InitializeBackgroundTasks()`. If you're uncomfortable using `async void`, I recommend using the [`SafeFireAndForget` extension method](https://github.com/brminnick/AsyncAwaitBestPractices#safefireandforget)